astropy/coordinates/angles/core.py

```diff
@@ -621,8 +621,7 @@ def _validate_angles(self, angles=None):
             # Otherwise, e.g., np.array(np.pi/2, 'f4') > np.pi/2 will yield True.
             angles_view = angles_view[np.newaxis]
 
-        invalid_angles = np.any(np.abs(angles_view) > limit)
-        if invalid_angles:
+        if np.any(np.abs(angles_view) > limit):
```

Comment:
Yes, better! (Could it be that the one before was just because of `black` annoyingly splitting lines?)

Reply:
No, the inline check was removed because of `numpy`, see bfac35635ffea23289645ad974952a4073052c25